apps/test/unit/storage/dataBrowser/dataVisualizer/VisualizerModalTest.js

```diff
@@ -143,4 +143,64 @@ describe('VisualizerModal', () => {
       ).to.deep.equal(expectedNumericColumns);
     });
   });
+
+  describe('getValuesForFilterColumn', () => {
+    let wrapper;
+    beforeEach(() => {
+      wrapper = shallow(<VisualizerModal {...DEFAULT_PROPS} />);
+    });
+
+    it('shows quotes around strings', () => {
+      let records = [
+        {id: 1, col: 'xyz'},
+        {id: 2, col: 'def'},
+        {id: 3, col: '123'},
+        {id: 4, col: 'abc'}
+      ];
+      expect(
+        wrapper.instance().getValuesForFilterColumn(records, 'col')
+      ).to.deep.equal(['"xyz"', '"def"', '"123"', '"abc"']);
```

Comment:
For those reading this test in the future, I'd remove some of the redundant examples from this test case.  Maybe just keep `'abc'` and `'123'`.